com.kms.cura_server/src/com/kms/cura_server/UserAPI.java

```diff
@@ -94,13 +94,32 @@ public String createDoctor(String jsonData) throws ClassNotFoundException, SQLEx
 		try {
 			DoctorUserEntity entity = JsonToEntityConverter.convertJsonStringToEntity(jsonData, getDoctorEntityType());
 			DoctorUserEntity user = DoctorUserDAL.getInstance().createUser(entity);
-			return SuccessResponse(user);
+			return SuccessResponse(user,DoctorUserEntity.DOCTOR_TYPE);
 		} catch (ClassNotFoundException | SQLException | DALException e) {
-			return UnsuccessResponse(e);
+			return UnsuccessResponse(e.getMessage());
 
 		}
 	}
 
+	@POST
+	@Path("/userLogin")
+	public String userLogin(String jsonData){
+		UserEntity entity = JsonToEntityConverter.convertJsonStringToEntity(jsonData, getPatientUserType());
+		try {
+			PatientUserEntity patientUserEntity=PatientUserDAL.getInstance().searchPatient(entity);
+			if (patientUserEntity != null){
+				return SuccessResponse(patientUserEntity,PatientUserEntity.PATIENT_TYPE);
```

Comment:
after re-factor for UserEntity then refactor this method for one return line only for both Patient and Doctor